contracts/DecentAutonomousAdmin.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IHats} from "./interfaces/hats/full/IHats.sol";
+import {IHatsElectionEligibility} from "./interfaces/hats/full/IHatsElectionEligibility.sol";
+import {ISablierV2Lockup} from "./interfaces/sablier/full/ISablierV2Lockup.sol";
+import {FactoryFriendly} from "@gnosis.pm/zodiac/contracts/factory/FactoryFriendly.sol";
+
+contract DecentAutonomousAdmin is FactoryFriendly {
+    string public constant NAME = "DecentAutonomousAdmin";
+    string public constant version_ = "0.1.0";
+
+    struct SablierStreamInfo {
+        uint256 streamId;
+        ISablierV2Lockup sablierV2Lockup;
+    }
+    struct TriggerStartArgs {
+        address currentWearer;
+        IHats userHatProtocol;
+        uint256 userHatId;
+        address nominatedWearer;
+    }
+
+    // //////////////////////////////////////////////////////////////
+    //                         initializer
+    // //////////////////////////////////////////////////////////////
+    function setUp(bytes memory initializeParams) public override initializer {}
```

Comment:
Uses the same patterns as the modules deployed via `ModuleProxyFactory` which requires a setUp function with params, but in its current state we don't need it